cerberus-cli/src/main/java/ca/on/oicr/gsi/cerberus/cli/Configuration.java

```diff
@@ -1,11 +1,13 @@
 package ca.on.oicr.gsi.cerberus.cli;
 
+import java.util.ArrayList;
 import java.util.Map;
 
 public final class Configuration {
 
   private Map<String, PineryConfiguration> pinery;
   private Map<String, String> vidarr;
+  private ArrayList<String> ignore;
```

Comment:
Should be `List` here, with the more specific `ArrayList` used when instantiating.